package.json

```diff
@@ -83,9 +85,13 @@
         "react-test-renderer": "^17.0.2",
         "rollup-plugin-inline-svg": "^2.0.0",
         "size-limit": "^7.0.5",
+        "storybook-dark-mode": "^1.0.8",
         "styled-components": "^5.3.3",
         "tsdx": "^0.14.1",
         "tslib": "^2.3.1",
         "typescript": "^4.5.4"
+    },
+    "dependencies": {
+        "web3uikit": "^0.1.16"
```

Comment:
Hi dude, why would the kit be a dependency of itself?

Reply:
Good question, I will remove. Good catch!